examples/tokenvm/cmd/token-cli/cmd/action.go

```
@@ -138,6 +167,17 @@ var createAssetCmd = &cobra.Command{
 			return err
 		}
 		printStatus(tx.ID(), success)
+
+		if supply > 0 {
+			// Generate transaction
+			err = executeAndWaitForMintAssetTransaction(ctx, cli, tx.ID(), actor.PublicKey(), supply, factory)
```

This error is ignored

---

done